src/api/erc20/Erc20Api.ts

```diff
@@ -229,6 +262,12 @@ export class Erc20ApiImpl implements Erc20Api {
 
     return (Erc20ApiImpl._contractsCache[networkId][address] = newERC20)
   }
+
+  private _getLocalErc20Property<P extends keyof Erc20Props>(address: string, prop: P): Erc20Props[P] | undefined {
+    const erc20Details = this.localErc20Details[address]
+
+    return erc20Details ? erc20Details[prop] : undefined
```

Comment:
Please try
```ts
return this.localErc20Details[address]?.[prop]
```
I just want to see if it works. Don't actually use it, not very readable.

Reply:
doesn't work i was trying something similar in my code for usePending

Reply:
Oh well /shrug